app/controllers/api/v1/projects/enabled_dynamic_fields_controller.rb

```diff
@@ -0,0 +1,43 @@
+module Api
+  module V1
+    module Projects
```

Comment:
At some point we'll need to add `include_examples 'requires user to have correct permissions' do` and check for project admin permissions, but that can come later.